Openshift-EE/pipelines/OpenEBS-base/stages/8-csi-functional/TUZX-Local-Backup-Restore/local-backup-restore

```diff
@@ -0,0 +1,129 @@
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:local-backup-restore-csi $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id 4H27 8-csi-functional "Create backup and restore and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 2 #
+################
+
+echo "*******Performing Backup and Restore on Target**********"
+
+run_id="local-csi";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=velero-backup-restore metadata=${run_id})
+echo $test_name
+
+cd litmus
+cp experiments/functional/backup_and_restore/run_litmus_test.yml run_backuprestore_test_cstor_csi_local.yml
+
+sed -i -e 's/value: app-percona-ns/value: percona-backup-csi-local/g' \
+-e 's/app: velero-backup-restore/app: velero-local-backup-restore-csi/g' \
+-e 's/value: "cstor"/value: "cstor-csi"/g' \
+-e '/name: DEPROVISION_VELERO/{n;s/.*/            value: "true"/}' \
+-e '/name: LOCAL_SNAPSHOT/{n;s/.*/            value: "true"/}' run_backuprestore_test_cstor_csi_local.yml
+
+sed -i "s/v1.1.0/$velero_version/g" run_backuprestore_test_cstor_csi_local.yml
+sed -i "s/plugin:ci/plugin:$velero_plugin_version/g" run_backuprestore_test_cstor_csi_local.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' run_backuprestore_test_cstor_csi_local.yml
+
+echo "Running the litmus test for Percona Backup Restore.."
+cat run_backuprestore_test_cstor_csi_local.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:velero-local-backup-restore-csi' job=run_backuprestore_test_cstor_csi_local.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:local-backup-restore-csi $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+rc_val=$(echo $?)
+current_time=$(eval $time)
+
+if [ "$rc_val" != "0" ]; then
+bash Openshift-EE/utils/e2e-cr jobname:local-backup-restore-csi jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag" test_result:Fail
+python3 Openshift-EE/utils/result/result_update.py $job_id 4H27 8-csi-functional "Create backup using non-default S3 profile and restore and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+bash Openshift-EE/utils/e2e-cr jobname:local-backup-restore-csi jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag" test_result:Pass
+python3 Openshift-EE/utils/result/result_update.py $job_id 4H27 8-csi-functional "Create backup using non-default S3 profile and restore and verify if the application is not impacted" Pass $pipeline_id "$current_time" $commit_id $gittoken
```

Comment:
Include readme in master branch for this test

Reply:
@nsathyaseelan  PTAL: https://github.com/mayadata-io/e2e-openshift/pull/395